packages/notebook/src/actions.tsx

```diff
@@ -940,13 +1031,7 @@ export namespace NotebookActions {
     if (!notebook.model || !notebook.activeCell) {
       return;
     }
-
-    const state = Private.getState(notebook);
-
-    notebook.mode = 'command';
-    notebook.model.cells.redo();
-    notebook.deselectAll();
-    Private.handleState(notebook, state);
+    // TODO
```

Comment:
Undo/Redo is not yet implemented in the datastore. This is an important blocker for this to be complete.